test/query_members.js

```diff
@@ -47,19 +46,13 @@ describe('Query Members', function() {
 		await ssClient.updateUser({ id: pending, name: 'Carlos' });
 		await ssClient.updateUser({ id: rejected, name: 'Joseph' });
 		await ssClient.updateUser({ id: banned, name: 'Evil' });
-		await createUsers([mod, rob, adam]);
 
-		csClient = await getTestClientForUser(rob);
 		channel = ssClient.channel('messaging', uuidv4(), {
 			created_by_id: mod,
 		});
 		await channel.create();
 		await channel.addModerators([mod]);
-		await channel.addMembers([rob]);
-		await channel.addMembers([rob2]);
-		await channel.addMembers([adam]);
-		await channel.addMembers([banned]);
-
+		await channel.addMembers([rob, rob2, adam, banned]);
```

Comment:
this is on purpose.
our created_at has microsecond precision, and created_at is assigned in a loop.
we need this to ensure consistent test results

Reply:
yup, reverted it to be explicit.